global.json

```diff
@@ -1,7 +1,8 @@
 {
-  "sdk": {
-    "allowPrerelease": true
-  },
+    "sdk": {
+        "allowPrerelease": true,
+        "version": "7.0.100-rc.2.22477.23"
+    },
```

Comment:
Sorry I didn't catch this! Can you please revert this change for this PR?

Reply:
Sure! Removed now 👍🏻